proxy_docker/app/script/manage_missed_conf.sh

```diff
@@ -111,7 +111,8 @@ manage_missed_conf() {
 
       latesttxid=$(echo "${received_address}" | jq -r ".txids | last")
       trace "[manage_missed_conf] latesttxid=${latesttxid}"
-      data='{"method":"gettransaction","params":["'${latesttxid}'"]}'
+      data="{\"method\":\"gettransaction\",\"params\":[\"${latesttxid}\",true]}"
```

Comment:
Interesting, watching01.dat and xpubwatching01.dat are not created as watch-only wallets, so the "true" arg wasn't needed... But I think it's a good idea to add it, just in case.  After all, this is a watching wallet...